include/polycap-photon.h

```diff
@@ -107,6 +138,9 @@ polycap_vector3 polycap_photon_get_exit_direction(polycap_photon *photon);
 POLYCAP_EXTERN
 polycap_vector3 polycap_photon_get_exit_electric_vector(polycap_photon *photon);
 
+POLYCAP_EXTERN
+void polycap_photon_get_extleak_data(polycap_photon *photon, polycap_leak **leaks, int64_t *n_leaks, polycap_error **error);
```

Comment:
Deze method zou een bool moeten returnen om aan te geven of er een probleem was of niet. Gebruik van de error is optioneel in C!